### ISSUE TITLE
MIssing support for FW v0.6.1

### ISSUE THREAD

**Issue by gkujawsk** [barakwei/ParticlePapertrail#4]
Hello,
I successfuly used ParticlePapertrail with v0.6.0 release. Today I gave a try with v0.6.1 and compiler complains that 'Filters' does not name a type. I tried to look around the firmware code and they refer to LogCategoryFilters instead of Filters. In fact I wasn't able to found the Filters definition. I changed Filters to LogCategoryFilters and compiler stoped to complain. It would be good to someone review this.  

Here's the exact error message:
```
../../dot11/papertrail.h:18:114: error: 'Filters' does not name a type
     explicit PapertrailLogHandler(String host, uint16_t port, String app, LogLevel level = LOG_LEVEL_INFO, const Filters &filters = {});
```

**Comment by barakwei** [comment]
I'm guessing you looked at the latest develop, and not in the 0.6.0 branch. The definition is [here](https://github.com/spark/firmware/blob/release/v0.6.0/wiring/inc/spark_wiring_logging.h). See the actual change this one https://github.com/spark/firmware/commit/cbda226b50530947884d80c83cc047b705e61645#diff-5ebce927d3fbd8c4ea993ca6a65f673f

**Comment by barakwei** [comment]
Once the official 0.6.1 is out, I'll update the library.

**Comment by gkujawsk** [comment]
Actually v0.6.1 is out there already :) Thanks for the pointer!
It's still not available on web ide, but it can be downloaded from the git
repository.

-- 
/G

**Comment by barakwei** [comment]
Good to know. I'll wait for web IDE release, I'm guessing this will be real soon.

**Comment by barakwei** [comment]
It's now in the web IDE, will update soon.

**Comment by barakwei** [comment]
Fixed in  284f0f0. Uploaded version 0.0.3 to Particle.
